src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,10 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireCC0 = () => {
+  const copyrightRegex = /^[\s\S]*\n## Copyright\n{1,2}Copyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
+  if (context.payload?.pull_request?.body && !(copyrightRegex.test(context.payload?.pull_request?.body))) {
```

Comment:
TypeScript should be smart enough to figure out that we have already handled the falsy cases, so we don't need the null coalescing operator after the `&&`.
```suggestion
  if (context.payload?.pull_request?.body && !(copyrightRegex.test(context.payload.pull_request.body))) {
```